app/renderer/js/pages/preference/new-server-form.ts

```diff
@@ -17,11 +17,16 @@ export function initNewServerForm({$root, onChange}: NewServerFormProps): void {
     <div class="server-input-container">
       <div class="title">${t.__("Organization URL")}</div>
       <div class="add-server-info-row">
-        <input
-          class="setting-input-value"
-          autofocus
-          placeholder="your-organization.zulipchat.com or zulip.your-organization.com"
-        />
+        <label class="setting-input-value">
+          <input
+            class="setting-input-add-server"
+            autofocus
+            placeholder="your-organization.zulipchat.com"
+            style="width: 34ch"
```

Comment:
Can you do all the styles in CSS files? It's generally harder to read code split between places.

Reply:
That makes sense, I went ahead and made this change in my most recent commit 

My original thought was that by having this specific rule inline it would be contextual and less of a magic number. 34ch is the initial width of the input field which is the width of the placeholder text 

Maybe I should add a comment in the stylesheet? or perhaps it's obvious